src/api/Tag.php

```diff
@@ -8,6 +8,11 @@ class Tag
     const ENV = 'env';
     const SPAN_TYPE = 'span.type';
     const SPAN_KIND = 'span.kind';
+    const SPAN_KIND_VALUE_SERVER = 'server';
+    const SPAN_KIND_VALUE_CLIENT = 'client';
+    const SPAN_KIND_VALUE_PRODUCER = 'producer';
+    const SPAN_KIND_VALUE_CONSUMER = 'consumer';
+    const SPAN_KIND_VALUE_INTERNAL = 'internal';
```

Comment:
This can be controversial (so far we have used `Tag.php` for keys only. I do not consider it a problem, but happy to revert back if the reviewer is not happy with this approach.

Reply:
I think it's fine :-)